Core/clim-core/presentation-defs.lisp

```diff
@@ -1518,6 +1518,7 @@ protocol retrieving gestures from a provided string."))
   (frob real)
   (frob rational)
   (frob ratio)
+  (frob integer)
```

Comment:
I think you made a type in the commit message.

Reply:
Fixed.